src/test_l1_dsDisplay.c

```diff
@@ -502,8 +503,22 @@ void test_l1_dsDisplay_positive_dsGetEDID(void) {
             UT_ASSERT_EQUAL(result, dsERR_NONE);
```

Comment:
We should not use setting header files and setting variables. We need to get device configurations from profile yaml file. Remove kSupportedPortTypes

Reply:
done